docs/build-repro.md

```diff
@@ -23,6 +23,15 @@ The structure is base64 encoded and has the following format when decoded:
 
 ```json
 {
+  "frontend": "dockerfile.v0",
+  "attrs": {
+    "build-arg:foo": "bar",
+    "cmdline": "crazymax/dockerfile:master",
+    "context": "https://github.com/crazy-max/buildkit-buildsources-test.git#master",
```

Comment:
something to think about: would be good if we could capture if build used a local context or no context. That is important info for replay.

Reply:
If a local context is being used then `context` field is not set:

```
"attrs": {
  "cmdline": "crazymax/dockerfile:buildattrs",
  "filename": "Dockerfile",
  "source": "crazymax/dockerfile:buildattrs"
}
```

Do you think it's worth adding something like `"context": "local"`?

Reply:
> If a local context is being used then context field is not set:

Yes but it might also be that there is no local context at all. Where you just build from Dockerfile with no `COPY` commands from context. On replay, we should be able to give an error when the local source is not passed by the user and required for replay, but not when it is not required at all.